package.json

```diff
@@ -5,18 +5,21 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "NODE_ENV=development nodemon server.js",
```

Comment:
the test line should be something like 

```json
"scripts": {
    "test": "mocha ./**/*.test.js",
    "dev": "NODE_ENV=development nodemon server.js",
    "start": "nodemon server.js"
  },
```

Then you can run them with `npm test`